packages/react-components/src/components/iot-app-kit-config/defaultValues.ts

```diff
@@ -0,0 +1,9 @@
+import { FEATURE_FLAGS } from './types';
```

Comment:
should the feature flags and the default value be defined in dashboard? I think the config can just be defined here as an object with boolean values?